source/modules/collection-api/sale/sale_items.md.erb

```diff
@@ -0,0 +1,20 @@
+### Sale Items
+
+```shell
+{
+    "item_id":           "101",
+    "value":             25.00,
+    "event_id":          23553,
+    "event_session_ids": ["101", "102"]
+}
+```
+
+Our current collection API is built with certain assumptions about the ticketing industry. In particular, we assume that a sale will have one or more items (in the case of the ticketing industry, tickets) associated with it. We require some information about these items in order to reason about the sale:
+
+Property        | When is it required?  | Type | Description
+--------        | --------------------- | ---- | -----------
+**id**          | `POST`, `UPDATE`      | String  | A unique identifier for this item (*e.g. ticket*) within your platform
+**event_id**    | `POST`                | String  | The unique identifier of the [event](#event-activity) that the item is tied too
+**session_ids** | `POST`                | Array   | The array of [event datetime IDs](#event-datetimes) that the ticket is tied to
+**price**       | `POST`                | Double  | The price of the item. *It should only contain numbers, dots, or commas*
+**quantity**    | `POST`                | Integer | The number of item of this type being bought.
```

Comment:
Are we still going with quantity here? This impacts the way we're dealing with transfers, as we'd have to specify how many tickets were sent to another user. 

Having a single ticker per item would make things a lot easier.

Reply:
Two things: I think when we're updating an item of a sale, we need to receive within the sale object all the items associated with the sale, even if it hasn't been updated. Otherwise, if a sale had two items, for example, and we received a sale update request with only the item the client wants to update, in our DB, when filtering out through update_timestamp, we'd see that that sale has only one item associated with, since the latest update timestamp would contain only the updated item. (I'm not sure if updating items information would be a reasonable thing to do, but as it can be done, I think we have to get it clear.)

Another thing I realized is that we might have sparse rows in our end tables in case clients update things a lot. Since we're in append mode only, when we receive an update, I suppose we're just gonna insert the new row with the values sent to us through the client., meaning all the other ones will be empty. Right?

Reply:
> Having a single ticker per item would make things a lot easier.\

Yup, my b--this is a typo

First point of the second comment: yes, good point! I believe we had discussed this too. I'll make sure to add a note :) 

Second point of second comment: not sure I follow? The situation you described is correct, but not sure I see the problem?

Reply:
Oh, I didn't mean it as a problem, I was just wondering about how we'd query data. Would probably have to go with something like latest `update_timestamp` where wanted value is `not null`.

Reply:
yup, exactly--that's actually what we do here https://github.com/inspetor/eval-txn-worker/blob/master/src/query_helpers/query_generator.py#L11